app/views/posts/index.html.erb

```diff
@@ -1,2 +1,45 @@
-<h1>List of all posts given by a user</h1>
+<div class="card mb-3 text-center d-flex justify-content-center" style="max-width: 540px;">
+  <div class="row g-0 text-center">
+    <div class="col-md-4">
+      <img src="<%= @user.photo %>" class="img-fluid rounded-start" alt="profile">
+    </div>
+    <div class="col-md-8">
+      <div class="card-body">
+        <h4 class="card-title"><%= @user.name %></h4>
+        <p class="card-text"><h6>Number of posts: <%= @user.post_counter %></h6></p>
+      </div>
+    </div>
+  </div>
+</div>
+<% @posts.each do |post| %>
+<%= link_to user_post_path(user_id: @user.id, id: post.id), class:"nav-link mt-4 d-flex justify-content-center" do %>
+<div class="card mb-3 text-center" style="max-width: 540px;">
+  <div class="row g-0 text-center">
+    <div class="col-md-8">
+      <div class="card-body">
+      <h2 class="card-title"><%= post.title %></h2>
+        <h5 class="card-title"><%= post.text %></h5>
+        <p class="card-text"><span>Comments: <%= post.comment_counter %></span></p>
+      </div>
+    </div>
+        <% end %>
+    <h6>Recent Comments</h6>
+    <% if post.recent_comments.empty? %>
+     <div class="card mb-3 border-secondary" style="width: 540px;">
+       <div class="card-body">
+            <p class="card-text">No Comments</p>
+          </div>
+        </div>
+ <% else %>
+     <div class="card mb-3 border-secondary" style="width: 540px;">
+          <div class="card-body">
+            <% post.recent_comments.each do |comment| %>
+              <p class="card-text"><%= comment.author.name %>: <%= comment.text %></p>
+              <% end %>
+          </div>
+      </div>
+
+
+ <% end %>
```

Comment:
- [ ] Kindly style the layout to match the wireframe in the post index page. The items should be centered while the text within the boxes should left-aligned as we have [in the design](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAACD3NNQDHEITUS5OBEHA4L4CZEISBOQ).

  ![image](https://github.com/enis-memic/blog-app/assets/81468675/76a478c6-e42e-4a2c-8813-10d4983f5ab4)